src/main/java/racinggame/domain/Cars.java

```diff
@@ -0,0 +1,43 @@
+package racinggame.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+
+    private final List<Car> cars;
+    private final int raceCount;
+
+    public Cars(int carCount, int raceCount) {
+        this.cars = initialCars(carCount);
+        this.raceCount = raceCount;
+    }
+
+    private static List<Car> initialCars(int carCount) {
+        List<Car> cars = new ArrayList<>();
+        for (int i = 0; i < carCount; i++) {
+            cars.add(new Car());
+        }
+        return cars;
+    }
+
+    public void raceStart() {
+        for (int i = 0; i < raceCount; i++) {
+            runCars();
+        }
+    }
+
+    private void runCars() {
+        for (int i = 0; i < cars.size(); i++) {
+            cars.get(i).run(new MoveCondition());
+        }
```

Comment:
`enhanced for` 문으로 변경하면 `i`와 같은 불필요한 변수의 사용을 생략할 수 있을 것 같네요.
```suggestion
        for (Car car : cars) {
            car.run(new MoveCondition());
        }
```